lib/test_queue/iterator.rb

```diff
@@ -64,6 +65,7 @@ def each
       end
     rescue Errno::ENOENT, Errno::ECONNRESET, Errno::ECONNREFUSED
     ensure
+      $0 = procline
```

Comment:
I could set it to that while recording statistics, then to the default after that. Setting it back to the default as we exit this method seems like the right thing to do; once this method returns we are no longer running tests or recording stats etc.